File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        res = []
        n = len(temperatures)

        for i in range(n):
            j = i+1
            seen = False
            while j < n:
                if temperatures[j] > temperatures[i]:
                    res.append(j - i)
                    seen = True
                    break
                j += 1
            if not seen:
                res.append(0)
        
        return res

File: Data Structures & Algorithms/counting-bits/submission-0.cpp
class Solution {
public:
    vector<int> countBits(int n) {
        vector<int> res;
        for(int i=0;i<=n;i++){
            int c=0;
            for(int j=0;j<32;j++){
                if(i&(1<<j)) c++;
            }
            res.push_back(c);
        }
        return res;
    }
};
